### PR TITLE
Develop framework

### PR DESCRIPTION
**Pull request by ViktoriiaLozova**

### CODE REVIEW

File: serenity-automation-mentoring/src/test/java/serenity/EnvironmentPropertyLoader.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+    private EnvironmentPropertyLoader() {
+        try {
+            if (System.getProperty(ENVIRONMENT_CONFIG_SYSTEM_PROPERTY) == null) {
+                this.loadProperties(DEFAULT_CONFIG);
+            } else {
+                this.loadProperties("environments/" + System.getProperty(ENVIRONMENT_CONFIG_SYSTEM_PROPERTY));
+            }
+
+        } catch (IOException var2) {
```
Comment (AntonApanasovych) [inline serenity-automation-mentoring/src/test/java/serenity/EnvironmentPropertyLoader.java]: name var2 is taken from de-compiled code. Please use meaningful namings. For exceptions it is `e` or `ex`

File: serenity-automation-mentoring/src/test/java/serenity/bdd/pages/PetStorePage.java [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+import io.restassured.response.Response;
+import io.restassured.specification.RequestSpecification;
+import net.serenitybdd.rest.SerenityRest;
+import net.thucydides.core.annotations.Step;
+import org.apache.http.HttpHeaders;
+import serenity.Config;
+import serenity.models.Pet;
+
+public class PetStorePage {
```
Comment (AntonApanasovych) [inline serenity-automation-mentoring/src/test/java/serenity/bdd/pages/PetStorePage.java]: This is not a Page, use meaningful namings, i.e. `PetStoreRestSteps` or `PetStoreEndpoint`

File: serenity-automation-mentoring/src/test/java/serenity/bdd/steps/DefinitionSteps.java [hunk 3]
```diff
@@ -59,5 +58,24 @@ public void whenTheUserLooksUpTheDefinitionInTable(ExamplesTable wordTable) {
...
+    public void findPetByStatus(String status) {
+        endUser.find_by_status(status);
+    }
+
+    @When("delete pet by current id")
+    public void deletePetByCurrentId() {
+        endUser.delete_by_current_id();
+    }
 
+    @Then("check response code")
```
Comment (AntonApanasovych) [inline serenity-automation-mentoring/src/test/java/serenity/bdd/steps/DefinitionSteps.java]: Make this method parametrized, `check response code $code`

File: serenity-automation-mentoring/src/test/java/serenity/EnvironmentPropertyLoader.java (hunk 1 above)
Comment (ViktoriiaLozova) [reply serenity-automation-mentoring/src/test/java/serenity/EnvironmentPropertyLoader.java]: Done

File: serenity-automation-mentoring/src/test/java/serenity/bdd/pages/PetStorePage.java (hunk 2 above)
Comment (ViktoriiaLozova) [reply serenity-automation-mentoring/src/test/java/serenity/bdd/pages/PetStorePage.java]: Done

File: serenity-automation-mentoring/src/test/java/serenity/bdd/steps/DefinitionSteps.java (hunk 3 above)
Comment (ViktoriiaLozova) [reply serenity-automation-mentoring/src/test/java/serenity/bdd/steps/DefinitionSteps.java]: Done

**Pull request merged**